app/models/lightning.py

```diff
@@ -950,7 +950,10 @@ class LnInfo(BaseModel):
     commit_hash: str
 
     # The identity pubkey of the current node.
-    identity_pubkey: str
+    identity_pubkey: str = Query("the nodes pubkey")
+
+    # The complete URI (pubkey@physicaladdress:port) the current node.
+    identity_uri: str = Query("the nodes comnplete URI")
 
```

Comment:
"comnplete" :P If you are using vscode I can recommend the "Code Spell Checker" extension.

Reply:
comitted